docs/network_scenarios.md

```diff
@@ -0,0 +1,25 @@
+### Network scenario
+Scenario to introduce network latency, packet loss, bandwidth restriction in the Node's hostnework interface. The purpose of this scenario is to observe faults caused by random variations in the network.
+
+##### Sample scenario config
+```
+network_scenarios:                                  # Scenario to create an outage by simulating random variations in the network.
+  duration: 300                                     # seconds
+  node_name:                                        # node on which scenario has to be injected;
```

Comment:
Maybe allow passing a list of nodes to target? Thoughs?

Reply:
will take a comma separated sting as input , similar to node_scenarios_example.yml